ntp-udp/src/raw_socket.rs

```diff
@@ -253,14 +251,14 @@ mod recv_message {
         };
 
         if mhdr.msg_flags & libc::MSG_TRUNC > 0 {
-            warn!(
+            tracing::info!(
                 max_len = packet_buf.len(),
                 "truncated packet because it was larger than expected",
             );
```

Comment:
This is remote triggerable, and not necessarily a bug on our end, so suggest we make this debug.